packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1093,11 +1093,11 @@
       <shortDescription>No related items</shortDescription>
     </labels>
     <labels>
-      <fullName>quantic_OpensInSalesforceSubTab</fullName>
-      <value>Opens in a Salesforce console subtab</value>
+      <fullName>quantic_OpensInSubTab</fullName>
+      <value>Opens in a subtab</value>
       <language>en_US</language>
       <protected>false</protected>
-      <shortDescription>Opens in a Salesforce console subtab</shortDescription>
```

Comment:
in this label we have a problem with the word `console`. because when we have a Salesforce it doesn't necessary mean that we are in a Salesforce console, we can be in a Salesforce community.
I would suggest to use the value `Opens in a Salesforce sub tab`.